@navikt/core/react/src/timeline/period/NonClickablePeriod.tsx

```diff
@@ -36,9 +37,9 @@ const NonClickablePeriod = ({
         [direction]: `${left}%`,
       }}
     >
-      <span className="navds-timeline__period--inner">
+      <span className={cn("navds-timeline__period--inner")}>
         {icon}
-        <span className="sr-only">
+        <span className={cn("sr-only")}>
```

Comment:
hmm, maybe i was a bit overzealous in this semi-automatic process... though it shouldn't matter much (renaming does nothing if it doesn't match?... will waste a few more CPU cycles though... I think it might be easier to scan later for changes that hit classNames where there isn't any `navds` to rename and remove them then? 🤔 )